convert.go

```diff
@@ -110,8 +112,38 @@ func Move(oldPath string, newPath string, out io.Writer) error {
 		if err != nil {
 			return err
 		}
-		if inf.IsDir() || fn == SHARDING_FN || fn == README_FN {
-			// we are an empty directory or generated file so just remove it
+		if inf.IsDir() {
+			indir, err := os.Open(oldPath)
+			if err != nil {
+				return err
+			}
+			defer indir.Close()
+
+			names, err := indir.Readdirnames(-1)
+			if err != nil {
+				return err
+			}
+
+			for _, n := range names {
+				p := filepath.Join(oldPath, n)
+				// part of unfinished write transaction
+				// remove it
```

Comment:
I am not 100% convinced this is a good thing to do.  The file could contain important data.  I could be wrong but this seams like the type of thing that would end up in the "lost+found" directory if this where a traditional Unix filesystem.  Maybe we could move the file into into a special directory.  @whyrusleeping what do you think.